nilearn/datasets/struct.py

```diff
@@ -423,6 +424,23 @@ def fetch_oasis_vbm(n_subjects=None, dartel_version=True, data_dir=None,
         description=fdescr)
 
 
+def fetch_surf_fsaverage(data_dir=None):
```

Comment:
We need a docstring here, ideally with a seealso pointing to fetch_surf_fsaverage5.

Reply:
Actually, based on the discussion to include fesaverage5 in here, forget the point about the seealso.